README.md

```diff
@@ -34,29 +34,44 @@ these common tasks.
 - [API Design Guidelines](.github/API_DESIGN.md)
 
 ## Installation
+KerasCV supports both Keras 2 and Keras 3. We recommend Keras 3 for all new 
+users, as it enables using KerasCV models and layers with JAX, TensorFlow and 
+PyTorch.
 
-To install the latest official release:
+### Keras 2 Installation
+
+To install the latest KerasCV release with Keras 2, simply run:
 
 ```
-pip install keras-cv tensorflow --upgrade
+pip install --upgrade keras-cv
```

Comment:
probably we should include `tensorflow` on this line, as keras-cv does not depend on tensorflow